src/components/UI/Form/AutoComplete/AutoComplete.module.css

```
@@ -47,3 +47,9 @@
   font-weight: 400;
   margin: 0 0 8px 16px;
 }
+
+@media (max-width:768px) {
```

```suggestion
@media screen and (max-width: 768px) {
```

Can we please update media queries to include type in all places?